pkg/gui/presentation/commits.go

```diff
@@ -441,8 +441,15 @@ func displayCommit(
 	}
 
 	authorFunc := authors.ShortAuthor
-	if fullDescription {
+	switch common.UserConfig.Gui.CommitAuthorFormat {
+	case "short":
+		authorFunc = authors.ShortAuthor
```

Comment:
Nitpick: this assignment is a no-op, it has this value already. To make this clearer, we might say
```go
	var authorFunc func(string) string
	switch common.UserConfig.Gui.CommitAuthorFormat {
	case "short":
		authorFunc = authors.ShortAuthor
	case "full":
		authorFunc = authors.LongAuthor
	default:
		authorFunc = lo.Ternary(fullDescription, authors.LongAuthor, authors.ShortAuthor)
	}
```

Reply:
Agree, @stefanhaller, fixed.

Regarding `alwaysUseAuthorInitials`, we seem to be on the same wavelength because I originally considered that functionality and setting name specifically. 🙂

As for the long names hardcoded to 17. I chose the `commitAuthorFormat` setting name for two reasons. The main reason was that if the implementation of the ShortAuthor changes, the setting name still works. The second reason is that we can potentially expand the allowed values along the lines of `commitAuthorFormat: '2,17'`, thus setting the lengths of the names in the small and expanded panels, respectively.
``

Reply:
> Agree, @stefanhaller, fixed.

Thanks! Merged. Looking forward to your next PR!

> The second reason is that we can potentially expand the allowed values along the lines of `commitAuthorFormat: '2,17'`, thus setting the lengths of the names in the small and expanded panels, respectively.

Awesome, I was considering exactly the same, but wasn't sure if it's intuitive enough. Since we are now two people coming up with the same suggestion, it must be somewhat natural at least.

Here's a PR that does this: #3665. I chose to make it `commitAuthorFormat: 'truncateTo:2,17'`, I find this a little more explicit. Would be great if you could have a look!

Reply:
@anikiforov After talking to Jesse about it, we decided we'd rather remove the `commitAuthorFormat` config again, and instead offer two properties for configuring the length in the normal and expanded view, as this gives a little more flexibility and is not as awkward as the `truncateTo:` syntax added in #3665. We hope that's ok with you! See #3688 for the new approach.